facilitator/README.md

```diff
@@ -44,6 +44,14 @@ AWS SNS/SQS support is experimental and has not been validated. To use it, pass
 
 To support new task queues, simply add an implementation of the `TaskQueue` trait, defined in `src/task.rs`. Then, add the necessary argument handling and initialization logic to `src/bin/facilitator.rs`.
 
+## Configuration
+
+### `aggregate-worker` thread count
+
+The `aggregate` and `aggregate-worker` subcommands can be configured to spawn multiple worker threads that will sum over batches in an aggregation in parallel before reducing each batch's sum into a sum part that is transmitted to the portal server. By default, a single thread is used. Set either the `--thread-count` command line argument or the `THREAD_COUNT` environment variable (e.g., in a Kubernetes `ConfigMap`) to an integer value to spawn the corresponding number of threads.
+
+To make best use of a multithreaded `aggregate-worker` in Kubernetes, make sure to configure the CPU and memory limits appropriately so that multiple cores can be allocated. For instance, a CPU request of `0.1` and a CPU limit of `3` might work well for four worker threads, keeping in mind that since Prio batch processing involves a lot of network I/O, we would not expect four worker threads on four cores to see consistently high CPU utilization.
+
```

Comment:
> Since `aggregate` is not really used in prod, does it need to support parallelism?

No, but it's still best if `aggregate` and `aggregate-worker` are as similar as possible. Plus this way we don't have to maintain a serial implementation as well as the parallel one.

>In Go, parallelism (i.e. number of OS threads) is automatically determined by the number of CPUs reported by the machine at runtime startup. Does it make sense to do that here to reduce the amount of configuration synchronization people need?

In this case, I don't think one thread per CPU is a good idea. Especially when the storage is something like S3, `facilitator` is heavily I/O bound (i.e. we spend a lot of time waiting on responses from various cloud services), so threads will spend a lot of time parked in `read` or `write` and thus can allow other threads to execute.